### DISCUSSION TITLE
Reduce size of Page down Page Up Touch areas

### DISCUSSION THREAD

**Discussion by bookteo** [plateaukao/einkbro#388] [Ideas]
Hey, in my hisense a9 the Touch areas are almost a third of the Screen. I think the areas could be much thinner. Often it interferes with the ability to click in Something one the Website. Apart from that o Love the Feature. 

Either make it customizable or just reduce the width of the area.

**Comment by BrilliantThings** [comment]
I agree. This would be terrific for smaller devices.
